public/style_editor.js

```diff
@@ -992,7 +992,7 @@ class CONFIG extends STYLE {
     var id = this.get_id();
     var ret = "";
     ret += "<span title='"+ note +"'>" + name + "</span>&lt;\n";
-    ret += "<div style='margin-left:1em'>\n";
```

Comment:
Everything else seems to be whitespace and comments.
Why sneak this change in there?